geode-core/src/main/java/org/apache/geode/internal/cache/AbstractRegionMap.java

```diff
@@ -388,9 +378,21 @@ public void close(BucketRegion bucketRegion) {
     if (logger.isDebugEnabled()) {
       logger.debug("Clearing entries for {} rvv={}", _getOwner(), rvv);
     }
-    LocalRegion lr = _getOwner();
+    final LocalRegion lr = _getOwner();
     RegionVersionVector localRvv = lr.getVersionVector();
-    incClearCount(lr);
+
+    boolean isBucketRegion = false;
+
+    if (!(lr instanceof HARegion)) {
+      CachePerfStats stats = lr.getCachePerfStats();
+      if (stats != null) {
+        if (lr.isUsedForPartitionedRegionBucket()) {
+          lr.getPartitionedRegion().getPrStats().incBucketClearCount();
```

Comment:
you should move the incBucketClearCount to the end. That way you inc the op count at the same time you inc the time stat

Reply:
ok.